package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "@textile/threads-client",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Textile's JS client for interacting with remote Threads.",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "dist/src/index.js",
```

Comment:
Is that right? That’s strange... we might want to fix that.

Reply:
Dunno, seems weird. Maybe something has changed with `npm` since I've been a heavy user, but this plus using prepare was the only way I was able to get the pack to include the built files.

Reply:
Happy to get fixed with a diff PR.